pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -53,6 +53,9 @@ public enum ServerMeter implements AbstractMetrics.Meter {
   PARTIAL_UPSERT_KEYS_NOT_REPLACED("rows", false),
   UPSERT_OUT_OF_ORDER("rows", false),
   DELETED_KEYS_TTL_PRIMARY_KEYS_REMOVED("rows", false),
+  TOTAL_KEYS_MARKED_FOR_DELETION("rows", false),
```

Comment:
These metrics only show up when the new metadata manager is used?

That seems a bit counter intuitive to me, because the names suggest that these should be emitted whenever deletion is enabled.

Reply:
`TOTAL_KEYS_MARKED_FOR_DELETION` and `DELETED_KEYS_WITHIN_TTL_WINDOW` can be emitted in both the metadata manager. `DELETED_TTL_KEYS_IN_MULTIPLE_SEGMENTS` is only possible in the new one.

Reply:
Can we emit the other two in the other metadata manager as well?